src/tabs/classes.scss

```diff
@@ -2,18 +2,15 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import './index';
 
-.iui-tabs-horizontal {
-  @include iui-tabs-horizontal;
-}
-
-.iui-tabs-borderless {
-  @include iui-tabs-borderless;
-}
-
-.iui-tabs-pill {
-  @include iui-tabs-pill;
-}
+.iui-tabs {
+  @include iui-tabs;
+  @include iui-tabs-default;
```

Comment:
Lets have a separate class for this so we don't need those long selectors to target default tabs.

Reply:
I've added `.iui-default` class.